source/_integrations/enocean.markdown

```diff
@@ -26,7 +26,7 @@ related:
 
 The [EnOcean](https://en.wikipedia.org/wiki/EnOcean) standard is supported by many different vendors. There are switches and sensors of many different kinds, and typically they employ energy harvesting to get power such that no batteries are necessary.
 
-The EnOcean integration adds support for some of these devices. You will need a controller like the [USB300](https://www.enocean.com/product/usb-300/) in order for it to work.
+The EnOcean integration adds support for some of these devices. You will need a controller like the [USB300](https://www.enocean.com/product/usb-300/) in order for it to work.  Not all controllers are supported by this integration.  You can compare controllers like FAM14, FGW14, FTS14KS and USB300 at this excellent [gateway overview](https://github.com/grimmpp/home-assistant-eltako/blob/main/docs/gateways/readme.md).
```

Comment:
As part of our integration quality scale, we aim to list the supported devices in our documentation.

Can we add it here?

For more information, see also: <https://developers.home-assistant.io/docs/core/integration-quality-scale/rules/docs-supported-devices>